back/scripts/enrichment/communities_enricher.py

```diff
@@ -16,10 +19,49 @@ def get_dataset_name(cls) -> str:
 
     @classmethod
     def get_input_paths(cls, main_config: dict) -> typing.List[Path]:
-        return [CommunitiesSelector.get_output_path(main_config)]
+        return [
+            CommunitiesSelector.get_output_path(main_config),
+            BaremeEnricher.get_output_path(main_config),
+        ]
 
     @classmethod
     def _clean_and_enrich(cls, inputs: typing.List[pl.DataFrame]) -> pl.DataFrame:
```

Comment:
```suggestion
    def clean_and_enrich(cls, inputs: typing.List[pl.DataFrame]) -> pl.DataFrame:
```

Utilisée comme méthode principale d’enrichissement et non une méthode privé. Je recommande de la renommer en clean_and_enrich pour plus de clarté :)

Reply:
Attention, cette méthode est héritée de `BaseEnricher`. Ça sortirai du scope de la PR.

Vu que cette PR est accepté depuis 3 semaines, je te propose de la rebase en l'état et d'effectuer tes propositions dans une PR à part. Qu'en penses-tu ?